themes/theme-gmd/pages/Page/components/Content/index.jsx

```diff
@@ -21,12 +24,28 @@ import connect from './connector';
  * @return {JSX}
  */
 function PageContent({ configs, pageId }) {
-  const center = (pageId === PAGE_ID_INDEX) ? (
-    <Logo key="center" />
-  ) : (
-    <AppBar.Title key="center" title={configs.title} />
+  let center = (
+    <Fragment key="center">
+      <Portal name={APP_BAR_CENTER_BEFORE} />
```

Comment:
You've added `APP_BAR_CENTER` portals here, but the within `ui-material/AppBar/Components/Center/index.jsx` the `center` component is wrapped again with those portals. So please just keep the actual centered components - `Logo` and `AppBar.Title`.

![app_bar_portals](https://user-images.githubusercontent.com/13087792/51377563-8450f880-1b0b-11e9-869c-3eafc72d7dc8.jpg)

Reply:
Done